requirements.txt

```diff
@@ -0,0 +1,23 @@
+# --------
+# requires
+# --------
+
+webob
+
+# --------------
+# tests_requires
+# --------------
+
+requests
+
+# -------
+# develop
+# -------
+
+pytest
+pytest-cov
+pip
+pylint
```

Comment:
In some other services projects, we use flake8 for linting, e.g.

  https://github.com/mozilla-services/tokenserver/blob/master/Makefile#L57

Are you particularly excited about pylint or would using flake8 be just as good?  I'm just thinking about minimizing the amount of change in other repos, assuming we push towards them all following a similar standard.

Reply:
Pylint is the heart of the boilerplate. Pylint is a code checker while flake8 is just a style checker. I'am working with Pylint since near about 10 years and over the time in 95% it points me to my mistakes. Its great, safes me a lot of time. Here in the boilerplate pylint is a part of the tox test configuration (https://github.com/return42/hawkauthlib/blob/six/tox.ini#L2). With a simple `make test` before you commit, a high source code quality is assured.

If Pylint is new for you, I recommend to give it a try. In short it will safe your time also :)

Reply:
OK, let's give pylint a shot, but with the caveat that I expect this may cause quite a bit of code churn in other repos.  But I'm always open to trying new things!